src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -2048,11 +2045,13 @@ internal static string GetTokString(TokKind kind)
         /// </summary>
         /// <param name="text">Expression text to format.</param>
         /// <param name="flags">Optional flags to customize the behavior of underlying lexer and parser. By default, expression chaining is enabled.</param>
+        /// <param name="features">Power Fx features.</param>
         /// <returns>Formatted expression text.</returns>
-        public static string Format(string text, Flags flags = Flags.EnableExpressionChaining)
+        public static string Format(string text, Flags flags = Flags.EnableExpressionChaining, Features features = null)

```

Comment:
>Features features = null [](http://example.com/codeflow?start=95&length=24)

Not a blocker, but does it need to be optional?

Reply:
It does not need to be but added for flexibility. I see that the API for Format and FormatUserDefinitions can we made a bit consistent and improved , created an issue to handle these cleanups - https://github.com/microsoft/Power-Fx/issues/2706